module/VuFind/src/VuFind/ILS/Driver/Aleph.php

```diff
@@ -567,6 +573,27 @@ public function init()
         if (isset($this->config['Catalog']['default_patron_id'])) {
             $this->defaultPatronId = $this->config['Catalog']['default_patron_id'];
         }
+
+        $this->addressMappings = $this->getDefaultAddressMappings();
+
+        if (isset($this->config['AddressMappings'])) {
+            foreach ($this->config['AddressMappings'] as $key => $val) {
+                $this->addressMappings[$key] = $val;
+            }
+        }
+    }
+
+    protected function getDefaultAddressMappings() {
+        return [
+            'barcode' => 'z304-address-5',
+            'fullname' => 'z304-address-1',
+            'address1' => 'z304-address-3',
```

Comment:
Looks like address-3 is getting used here twice, and address-2 is not used at all. Is that intentional?